flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -1359,6 +1362,9 @@
                     <NODE_ID>SWITCH_OVHD_EXTLT_RWY</NODE_ID>
                     <PART_ID>EXTLT_RWY</PART_ID>
                     <ANIM_NAME>SWITCH_OVHD_EXTLT_RWY</ANIM_NAME>
+                <!-- TODO Remove when fixed in Asobo base files -->      
+                    <TT_DESCRIPTION_ID>COCKPIT.TOOLTIPSV2.LIGHTING_SWITCH_LIGHT_TAXI_ACTION_SET</TT_DESCRIPTION_ID>
+                <!-- TODO Remove when fixed in Asobo base files -->      
```

Comment:
Duplicate comment?

Reply:
Ok, in order to not block QA further over a minor change I've just committed this small adjustment, please pull the changes.

Comments are generally understood to refer to the following line (or git blame can be used), so there's no need to bookend.